### DISCUSSION TITLE
PostmarketOS support

### DISCUSSION THREAD

**Discussion by Sai-desu** [Gusaroo/tuna#3] [General]
Hi , Gusaroo! I was so impressed by your work on TunaDE and wanted to know if you can make port for pmOS. Breefly, PostmarketOS is mobile-friendly (but not only mobile -- it implements support for tablets , ARM laptops e.t.c) operating system for ARM and x86 architectures (I highly advice to see their site). But its main problem -- short list of stable working DE for non-mainlined devices (with old kernels) and their most often choice is XFCE. (Phosh requires mainline kernel) . I think it would popularize your project among biggest mobile Linux project..... What do you think about that?

**Comment by gusaroo** [comment]
Tuna will work on any linux distro even if they are no packages for it in their repository as you could just compile the source code. So I suppose I’ve already made a port for PostmarketOS. I hope this answers your question.
